lib/shim/webframework-shim.js

```diff
@@ -869,10 +856,13 @@ function _recordMiddleware(shim, middleware, spec) {
     txInfo.errorHandled |= isErrorWare
 
     // Copy over route parameters onto the transaction root.
-    const params = shim.agent.config.high_security
+    let params = shim.agent.config.high_security
       ? null
       : spec.params.call(this, shim, fn, fnName, args, req)
 
+    // Route parameters are handled here, query parameters are handled in lib/transaction/index.js#_markAsWeb as part of finalization
+    params = shim.prefixRouteParameters(params)
```

Comment:
these attributes are always added to the middleware spans regardless of attributes.include.  This gives feature parity from before but may confuse users since they are on the request.parameters.route name space.  I know we talked about it in slack but I'm not starting to understand why these weren't prefixed.  I'm torn on the _right_ approach here but this is prob the best